src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -1804,13 +1813,21 @@ def process_maximum_cap_constraint(dict_values, group, asset, subasset=None):
                 logging.warning(message)
                 asset_dict[MAXIMUM_CAP][VALUE] = None
 
-            # adapt maximumCap of non-dispatchable sources
+            # adapt maximumCap and maximumAddCap of non-dispatchable sources
             if group == ENERGY_PRODUCTION and asset_dict[FILENAME] is not None:
                 asset_dict[MAXIMUM_CAP][VALUE] = (
                     asset_dict[MAXIMUM_CAP][VALUE] * asset_dict[TIMESERIES_PEAK][VALUE]
                 )
                 logging.debug(
                     f"Parameter {MAXIMUM_CAP} of asset '{asset_dict[LABEL]}' was multiplied by the peak value of {TIMESERIES}. This was done as the aimed constraint is to limit the power, not the flow."
                 )
+                asset_dict[MAXIMUM_ADD_CAP][VALUE] = (
+                    asset_dict[MAXIMUM_ADD_CAP][VALUE]
+                    * asset_dict[TIMESERIES_PEAK][VALUE]
+                )
```

Comment:
I was wondering - should we do the `TIMESERIES_PEAK` adjustment directly in `D0`/`D1`? Or introduce a `MAXIMUM_ADD_CAP_NORMALIZED` alike the `TIMESERIES_NORMALIZED`? Basically, the transformation is done for oemof, and the corrected value there does not reflect the `MAXIMUM_ADD_CAP` (kWp installed) anymore but the `MAXIMUM_ADD_CAP_NORMALIZED`

Reply:
So calculate the `MAXIMUM_ADD_CAP_NORMALIZED` in `C0` instead of replacing the `MAXIMUM_ADD_CAP` value as shown above? And using `MAXIMUM_ADD_CAP_NORMALIZED in `D0/D1` instead of `MAXIMUM_ADD_CAP` whenever e.g. a `TIMESERIES_NORMALIZED` is present in the `asset_dict`

Reply:
Yeah, what you did in 99b3d04 sounds right to me! I think it is not necessary to have both `maximumCap_normalized` and `maximumAddCap_normalized`, but it also does not hurt :)